server.js

```diff
@@ -3092,6 +3095,55 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+// GitHub release date integration.
+camp.route(/^\/github\/release-date\/([^\/]+)\/([^\/]+)\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+  var user = match[1];  // eg, qubyte/rubidium
```

Comment:
It's totally cool to use `const` in new code, though could you use it throughout?